CONTRIBUTING.md

```diff
@@ -1,70 +1,103 @@
-## Contributing
+# Contributing
 
 | Component | Build Status |
 | --------- | ------------ |
 | Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
 | Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
```

Comment:
Lets use the terminology client and management libraries and we should give a definition up top of how to determine one type of library from the other.